quokka/core/db.py

```diff
@@ -183,17 +183,14 @@ def page_set(self, *args, **kwargs):
         return self.content_set(*args, **kwargs)
 
     def block_set(self, *args, **kwargs):
-            kwargs.setdefault(
-                'sort',
-                self.app.theme_context.get(
-                    'BLOCK_ORDER_BY', [('title', -1)]
-                )
-            )
-            if not args:
-                args = [{'content_type': 'block'}]
-            elif isinstance(args[0], dict):
-                args[0]['content_type'] = 'block'
-            return self.content_set(*args, **kwargs)
+        kwargs.setdefault('sort', self.app.theme_context.get(
+            'BLOCK_ORDER_BY', [('title', -1)]
```

Comment:
Consider using BLOCK_ORDER_BY as constant

Reply:
its not a constant but a keyword; :-)